CorsixTH/Lua/entities/humanoids/patient.lua

```diff
@@ -962,7 +990,7 @@ end
 function Patient:updateDynamicInfo()
   local action_string = self.action_string or ""
   local info = ""
-  if self.going_home then
+  if self.going_home or self.going_die then
```

Comment:
While fine, just note that Patient:die does update dynamic info directly. Not sure why it doesn't use this function instead but that's an aside.